lib/pull_request.rb

```diff
@@ -125,14 +129,31 @@ def tell_dependency_manager_what_dependencies_are_allowed
   end
 
   def tell_dependency_manager_what_dependabot_is_changing
+    dependency_updates = commit_message.scan(/Updates `(\w+)` from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d)/)
```

Comment:
Should this have the trailing `+` to support more than one digit for the patch version? E.g. `1.1.10`.

```suggestion
    dependency_updates = commit_message.scan(/Updates `(\w+)` from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d+)/)
 ```